content/en/docs/concepts/security/api-server-bypass-risks.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Kubernetes API Server Bypass Risks
+description: >
+  Security architecture information relating to the API server and other components
+content_type: concept
```

Comment:
nit: if you can explicitly define a `weight` here for ordering the pages, that would be nice